input-border.scss

```diff
@@ -1,15 +1,8 @@
+@import 'bower_components/vui-colors/colors.scss';
 
-$vui-input-borderWidth: 1px !default;
-$vui-input-borderColor: #cccccc !default;
-$vui-input-borderStyle: solid !default;
-$vui-input-borderRadius: 3px !default;
+$vui-input-borderRadius: 8px !default;
```

Comment:
For button, I converted the radius to rem so that the proportion of element dimension to radius stays the same.  This was an assumption that I made.  Not sure if you want to do the same here... ?